dev/config.yaml

```diff
@@ -1,7 +1,7 @@
 security:
   readonly: false
   encryption_key: "=tW$56zytgB&3jN2E%7-+qrGZE?v6LCc"
-  enable_auth: true
+  enable_auth: false
```

Comment:
```suggestion
    enable_auth: true
```

Reply:
yes this is always changed when running the e2e locally, need to adjust that. I reverted it